### PR TITLE
Update data model

### PR DESCRIPTION
**Pull request by mwbenowitz**
This commit brings the epub storage/accessibility report generator into line with the established data model for the SFR data pipeline. This function was originally created before the format was standardized, and this presented problems for merging data coming from s3 and the accessibility report generator. 

The new format complies with the data model and allows for asynchronous updating of records. This is important as it allows for new records to be ingested and the more time consuming process of persisting the epub files to disk and generating a report can be run as independent processes.

### CODE REVIEW

File: config/dev.env [hunk 1]
```diff
@@ -4,7 +4,7 @@ AWS_S3_ENDPOINT=
 AWS_S3_EPUB_BUCKET=simplye-research-epubs
 
 AWS_KINESIS_ENDPOINT=
-AWS_KINESIS_STREAMNAME=sfr-epub-results-development
+AWS_KINESIS_STREAMNAME=sfr-db-ingest
```
Comment (gkallenberg) [inline config/dev.env]: we need to have the '-development' string applied to stream names as per our standards.

File: config/local.env [hunk 2]
```diff
@@ -4,7 +4,7 @@ AWS_S3_ENDPOINT=
 AWS_S3_EPUB_BUCKET=simplye-research-epubs
 
 AWS_KINESIS_ENDPOINT=
-AWS_KINESIS_STREAMNAME=sfr-epub-results-development
+AWS_KINESIS_STREAMNAME=sfr-db-ingest
```
Comment (gkallenberg) [inline config/local.env]: same as above

File: src/epubParsers.js [hunk 3]
```diff
@@ -52,35 +52,50 @@ exports.getBuffer = (stream) => {
   })
 }
 
-exports.epubStore = (fileName, itemID, updated, type, response) => {
+exports.epubStore = (partName, instanceID, updated, type, response, itemData, fileName) => {
   logger.info('Storing file in S3')
```
Comment (gkallenberg) [inline src/epubParsers.js]: you might want to add some identifier information to the log message here to distinguish which file is being stored.

File: test/handler.test.js [hunk 4]
```diff
@@ -234,17 +221,19 @@ describe('Handlers [index.js]', () => {
       let updated = '2019-01-01'
       let fileName = 'fileName'
       checkStub.resolves('dataObject')
+      explodeStub.resolves('success')
       bufferStub.resolves('bufferObject')
+      storeStub.resolves('succeess')
```
Comment (gkallenberg) [inline test/handler.test.js:226]: sp on 'succeess'

**Review by gkallenberg — COMMENTED** [review]
Just a few comments, but looks good overall.

**Comment by mwbenowitz** [comment]
Made the few comments that Greg made as well as added a better output block when we find an existing epub file. The metadata that is included with the file should still be returned in case that is what has been updated. The new format allows the DB manager to take that and update the relevant item record.

**Pull request merged**